### ISSUE TITLE
kwonly function name

### ISSUE THREAD

**Issue by hmaarrfk** [hmaarrfk/wabisabi#17]
@jni you wanted to propose a different name for this function?

**Comment by jni** [comment]
Yes, I don't know what though. =) `keyword_only` or `future_keyword_only` are my off-the-cuff suggestions. I think both are improvements, despite being longer.
